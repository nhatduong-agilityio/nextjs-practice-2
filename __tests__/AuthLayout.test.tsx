// Utils
import React from 'react'
import TestRenderer from 'react-test-renderer'

// App
import AuthLayout from '../app/(auth)/layout'

describe('Auth Layout render', () => {
  it('RootLayout should match snapshot', () => {
    const tree = TestRenderer.create(<AuthLayout>Test Content</AuthLayout>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
