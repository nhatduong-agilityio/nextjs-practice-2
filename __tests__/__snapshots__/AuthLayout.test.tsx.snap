// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Layout render RootLayout should match snapshot 1`] = `
<html
  lang="en"
>
  <body>
    <div
      className="css-4yppbv"
    />
    <main
      className="css-70hxw1"
    >
      Test Content
    </main>
    <span />
    <span
      hidden={true}
      id="__chakra_env"
    />
  </body>
</html>
`;
